ArcGISRuntimeSDKQt_CppSamples/Analysis/Geotriggers/Geotriggers.h

```diff
@@ -85,6 +85,7 @@ class Geotriggers : public QObject
   QUrl m_currentFeatureImageUrl;
   QMap<QString, Esri::ArcGISRuntime::ArcGISFeature*> m_featureQMap;
   QMap<QString, QUrl> m_featureAttachmentImageUrls;
+  Esri::ArcGISRuntime::AttachmentListModel* m_attachmentListModel = nullptr;
```

Comment:
To keep things tidy, I recommend putting this next to the other `Esri::ArcGISRuntime::` pointers above.

Reply:
Removed this as we are calling - `m_selectedFeature->attachments(false, false)`.